gapis/atom/group.go

```diff
@@ -25,6 +25,10 @@ import (
 	"github.com/google/gapid/core/math/sint"
 )
 
+// Subrange represents a fully qualified range underneath a particular command.
+// i.e. the Subcommand range extends from 0 -> [x, x, x]
+type SubRange []uint64
```

Comment:
Despite the usage treating this as a range, I feel this should be an index as it only points to a single sub-command. Probably should go into api as `api.SubCmdIndex`. We could then add on to `api.CmdID`:

```go
// SubIndex returns an index to a sub-command of the given command ID.
func (i CmdID) SubIndex(indices ... uint64) SubCmdIndex {
    return SubCmdIndex(append([]uint64{uint64(i)}, indices...))
}
```

And on to `SubCmdIndex`:
```go
// SubCmdIndex returns the identifier of the root command that holds this subcommand index.
func (i SubCmdIndex) Root() CmdID {
    return CmdID(i[0])
}
```